### PR TITLE
Announce merging of selinux-policy and selinux-policy-contrib

### PR DESCRIPTION
**Pull request by zpytela**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Merging selinux-policy-contrib repository with selinux-policy
+
+Currently, SELinux policy packages in Fedora use 2 repositories:
+base [1] and contrib [2].
+This division into two repos is merely a historical artifact and
+the main effect is make dealing with the repos more difficult.
+
+We work on a change to merge these repos into one, containing sources from both.
+All the changes would affect both the repos, branches rawhide, and future branches
+f34 and newer.
```
Comment (WOnder93) [inline README.md]: I would suggest to mark up the branch names and change word order a bit:
```
All the changes would affect both repos, `rawhide` branches, and future branches
`f34` and newer.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+There is no change for users.
+
+### How custom selinux-policy developers will be affected?
+No change for policy writing 
+other then where to look for for the previous contrib modules.
+
+Scripts, data, specfile, etc. in the dist git will be updated to use the new location.
+
+### How selinux-policy contributors will be affected?
+No change other then where to look for for the previous contrib modules and where to submit pull requests.
```
Comment (WOnder93) [inline README.md]: I would also mention that existing PRs against the contrib repo need to be rebased and re-opened against base (by the submitter) in order to be merged.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### How to submit pull requests?
```
Comment (WOnder93) [inline README.md]: "Where to" sounds more appropriate here (and below).

File: README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Use the base selinux-policy repository [3].
+
+### How to report issues?
+Use the base selinux-policy repository [4].
+
+### Backporting commits
+Commits to policy/modules-contrib needing backport to stable branches will be
```
Comment (WOnder93) [inline README.md]: I think this should be "policy/modules/contrib"

**Review by WOnder93 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zpytela) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (zpytela) [reply README.md]: Updated, but at a different place.

File: README.md (hunk 3 above)
Comment (zpytela) [reply README.md]: Updated.

File: README.md (hunk 4 above)
Comment (zpytela) [reply README.md]: Updated.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+It currently is just a source of confusion and makes dealing
+with SELinux policy repos more difficult.
+
+We work on a change to merge these repos into one, containing sources from both.
+All the changes would affect both repos, `rawhide` branches and future branches
+`f34` and newer.
+After the merge, when working in the rawhide branch, only the base repo would
+be used; the corresponding contrib branch would be archived and not used
+any longer. The contrib repo's commit history would be a part of the base repo.
+Stable branches (f33, f32, all older ones) would remain unchanged.
```
Comment (WOnder93) [inline README.md]: These branches should also be marked up for consistency.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+### How users will be affected?
+There is no change for users.
+
+### How custom selinux-policy developers will be affected?
+No change for policy writing other then where to look for modules, previously found in the contrib repo.
+
+Scripts, data, specfile, etc. in the dist git will be updated to use the new location for builds targeting rawhide or f34+.
+
+### How selinux-policy contributors will be affected?
+No change other then where to look for the previous contrib modules and where to submit pull requests.
```
Comment (WOnder93) [inline README.md:33]: s/then/than/ (sorry, noticed it only after a second look)

**Review by WOnder93 — APPROVED** [review]
Just two more cosmetic issues; feel free to merge after fixing those.

**Pull request merged**